packages/core/src/dialects/sqlite/query-interface.ts

```diff
@@ -293,4 +382,162 @@ export class SqliteQueryInterfaceTypeScript extends AbstractQueryInterface {
 
     return data;
   }
+
+  async getForeignKeyReferencesForTable(tableName: TableNameOrModel, options?: QueryRawOptions) {
```

Comment:
can you specify the return type of each of these methods explicitly?

Reply:
Updated now.